[DiagnosticsClient] Update APIs (#50376)
dotnet/docs: docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```

This may be thrown when the runtime responds with an error to a given command.

### ProfilerAlreadyActiveException

```csharp
public class ProfilerAlreadyActiveException : ServerErrorException
```

> This may be thrown when a profiler is already loaded into the target runtime and another attach is attempted.

Reviewer: ```suggestion
This exception is thrown when a profiler is already loaded into the target runtime and another attach is attempted.
```
